doc/api/n-api.md

```diff
@@ -50,6 +52,30 @@ which is located in the src directory in the node development tree:
 #include <node_api.h>
 ```
 
+This will opt into the default `NAPI_VERSION` for the given release of Node.js.
+In order to ensure compatibility with specific versions of N-API, the version
+can be specified explicitly when including the header:
+
+```C
+#define NAPI_VERSION 3
```

Comment:
Need to update this to account for experimental

Reply:
I think the example below accomplishes this.

Reply:
Yeah, I made that note for myself and it should be resolved now.